src/balloonpanel/balloonpanelview.js

```diff
@@ -159,6 +159,43 @@ export default class BalloonPanelView extends View {
 }
 
 /**
+ * A horizontal offset of the arrow tip from the edge of the balloon. Controlled by CSS.
+ *
+ *		 +-----|---------...
+ *		 |     |
+ *		 |     |
+ *		 |     |
+ *		 |     |
+ *		 +--+  |  +------...
+ *		     \ | /
+ *		      \|/
+ *	    >|-----|<---------------- horizontal offset
+ *
+ * @default 30
+ * @member {Number} module:ui/balloonpanel/balloonpanelview~BalloonPanelView.arrowHorizontalOffset
+ */
+BalloonPanelView.arrowHorizontalOffset = arrowHorizontalOffset;
```

Comment:
Why are you exposing these values if you're not reading it later on but use the consts? This means that if someone changes the theme and try to change these values it's not going to work.